src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -492,11 +492,11 @@ public Object onTraversalSuccess() {
                                                                                    final int batchSize, final int lowerSampleIndex) {
         final SortedMap<String, FeatureReader<VariantContext>> sampleToReaderMap = new TreeMap<>();
```

Comment:
Add explicit docs to the sample name map argument clarifying whether the sample names must match the names in the header.

Reply:
added